DataFormats/Detectors/FIT/FDD/include/DataFormatsFDD/ChannelData.h

```diff
@@ -0,0 +1,44 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef _FDD_CHANNEL_DATA_H_
+#define _FDD_CHANNEL_DATA_H_
+
+#include <array>
+#include <Rtypes.h>
+
+/// \file ChannelData.h
+/// \brief Container class to store values of single FDD channel
+/// \author micha.broz@cern.ch
+
+namespace o2
+{
+namespace fdd
+{
+
+struct ChannelData {
+
+  int mPMNumber = -1;       // PhotoMultiplier number (0 to 16)
+  float mTime = -1024;      // Time of Flight
+  short mChargeADC = -1024; // ADC sample
+  short mFEEBits = 0;       //Bit information from FEE
+  //0 Integrator, 1 DoubleEvent, 2 Event1TimeLost, 3 Event2TimeLost, 4 AdcInGate, 5 TimeTooLate, 6 AmpTooHigh, 7 EventInTrigger, 8 TimeLost
+
```

Comment:
Thanks, I think one should also define corresponding bit defintions,e.g.
enum Flags {Integrator=0x1<<0, DoubleEvent=0x1<<1, ... TimeLost=0x1<<8};

Reply:
Done